app/build.gradle

```diff
@@ -73,8 +74,13 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp:4.6.0"
     implementation "com.squareup.okhttp3:logging-interceptor:4.6.0"
 
+    //room (database) dependencies
+    implementation "androidx.room:room-runtime:2.2.5"
+    kapt "androidx.room:room-compiler:2.2.5"
+
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    annotationProcessor 'androidx.room:room-compiler:2.2.2'
```

Comment:
Jak dodałeś już `kapt "androidx.room:room-compiler:2.2.5"` to tę linijkę możesz już usunąć.